client/my-sites/upgrades/checkout-thank-you/google-voucher/terms-and-conditions.jsx

```diff
@@ -2,41 +2,81 @@
  * External dependencies
  */
 import React from 'react';
-import i18n from 'i18n-calypso';
+import { localize } from 'i18n-calypso';
 
-const googleTermsAndConditions = () => {
-	return(
-		<div>
-			<h3>{ i18n.translate( 'Terms and conditions for this offer:' ) }</h3>
-			<p>{ i18n.translate( 'In the below terms, “AdWords” may mean AdWords or AdWords Express, as appropriate.' ) }</p>
-			<ol>
-				<li className="google-voucher__terms-and-conditions">{ i18n.translate( 'Offer available to customers with a billing address in the United States only. One promotional code per advertiser.' ) }</li>
+/*
+ * Constants
+ */
+const expirationDate = 'October 31, 2017';
```

Comment:
At the moment we don't have a process except to contact our Google Rep a month out, and getting a new set of codes. I have a calendar item for 10/1/2017